docs/guides/backends.rst

```diff
@@ -0,0 +1,184 @@
+.. _backends_guide:
+
+==============
+Backends guide
+==============
+
+Backends are the interfaces used to run :ref:`Circuits <circuits_guide>` in Orquestra Core. In this guide we'll cover differences between types of backends (``QuantumBackend`` vs ``QuantumSimulator``, ``TrackingBackend``, ``SymbolicSimulator``), how to use backends, currently integrated backends, and creating your own backend (among other topics).
+
+
+General Backend Information
+===========================
+
+The function of a backend is to send instructions to quantum hardware or to a simulator. Since there are various types of quantum hardware and simulators, different sets of instructions needs to be written and it can become complex. ``QuantumBackend`` simplifies this process by enabling the user to deploy a single set of instructions on multiple hardware and simulators simultaneously. Therefore executing code on various devices becomes easier and more manageable.
+
+Two types of backends
+---------------------
+
+``QuantumBackend`` is the base class for all backends. ``QuantumBackend``\ s are used for physical quantum hardware backends.
```

Comment:
Well, that's not technically correct, as `QuantumSimulator` inherits from `QuantumBackend`.

```suggestion
``QuantumBackend`` is the base class for all backends. ``QuantumBackend``\ s can be used for physical quantum hardware backends as well as simulators.
```